src/components/NewDraftWizard/WizardHooks/WizardTransformTemplatesToDrafts.tsx

```diff
@@ -8,7 +8,7 @@ import { getObjectDisplayTitle, getOrigObjectType } from "utils"
 const transformTemplatesToDrafts = async (
   templateAccessionIds: Array<string>,
   templates: Array<ObjectInsideFolder>,
-  dispatch: (dispatch: (reducer: any) => void) => void
+  dispatch: (dispatch: (reducer: never) => void) => void
```

Comment:
Having type `never` for `reducer` is a bit unclear to me. Could `type { AppDispatch }` be possible in this case ?

Reply:
Good catch. `AppDispatch` type suits it a lot better.